middleware/middleware.go

```diff
@@ -19,3 +19,5 @@ func Chain(m ...Middleware) Middleware {
 		return next
 	}
 }
+
+type StreamMiddleware func(Handler) Handler
```

Comment:
It doesn't seem to be used

Reply:
I will remove it

Reply:
It seems like it hasn't been deleted yet